frontend/src/components/2DModel/Modelization.vue

```diff
@@ -46,6 +46,12 @@ import {
 } from "src/components/Monaco/svg_maths";
 import EventBus from "src/services/EventBus";
 
+function randomString({ length }) {
```

Comment:
Please squatch you second commit with the first.

directly give the length as parameters instead of Object that contains length.